src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -79,13 +64,32 @@ class CornerstoneViewport extends Component {
     startLoadHandler: PropTypes.func,
     endLoadHandler: PropTypes.func,
     loadIndicatorDelay: PropTypes.number,
+    loadingIndicatorComponent: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.func,
+    ]),
```

Comment:
I think we can use `PropTypes.element` here?
https://www.npmjs.com/package/prop-types

Reply:
Just element kept throwing an error in the console, so, made it one of element or function.